examples/geometry.shader_test

```diff
@@ -0,0 +1,104 @@
+# Copyright 2018 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Is there any particular reason you have chosen the Apache license rather than the MIT Expat license?

It would be good to add a [require] section with ‘geometryShader’ here. Otherwise if you run the test with the validation layers it will complain that the feature is not requested. (I guess you could argue that VkRunner should just always request that feature if there is a geometry shader present…)

Reply:
Changed it to MIT license.